rbac/core/kafka.py

```diff
@@ -35,15 +40,32 @@ class RBACProducer:
     def get_producer(self):
         """Init method to return fake kafka when flag is set to false."""
         if not hasattr(self, "producer"):
+            retries = 0
+            max_retries = 5
             if settings.DEVELOPMENT or settings.MOCK_KAFKA or not settings.KAFKA_ENABLED:
                 self.producer = FakeKafkaProducer()
+                logger.info("Fake Kafka producer initialized in development mode")
             else:
-                if settings.KAFKA_AUTH:
-                    self.producer = KafkaProducer(**settings.KAFKA_AUTH)
-                elif not settings.KAFKA_SERVERS:
-                    raise AttributeError("Empty servers list")
-                else:
-                    self.producer = KafkaProducer(bootstrap_servers=settings.KAFKA_SERVERS)
+                try:
+                    while retries <= max_retries:
+                        if settings.KAFKA_AUTH:
+                            self.producer = KafkaProducer(**settings.KAFKA_AUTH)
+                            logger.info("Kafka producer initialized successfully")
+                            break
```

Comment:
can we just use return here: 
return KafkaProducer(bootstrap_servers=settings.KAFKA_SERVERS)